src/errors.ts

```diff
@@ -44,10 +44,12 @@ function isConnectionError(err: Error): boolean {
   if (err instanceof SQSError) {
     return (
       err.statusCode === 403 ||
+      err.name === "SQSError" ||
```

Comment:
This casts a wide net although in some cases SQS wasn't returning an error code. This is difficult to test for with localstack as you can't easil mock different failures due to DNS resolution etc. This does guarantee though that you can't be caught in a loop with no pauses spamming any logging and network requests.